cupyx/_runtime.py

```diff
@@ -215,6 +220,7 @@ def __str__(self):
             ('CuPy Version', self.cupy_version),
```

Comment:
Let's add platform information too.

```suggestion
            ('CuPy Version', self.cupy_version),
            ('CuPy Platform', 'NVIDIA CUDA' if not is_hip else 'AMD ROCm'),
```